.github/workflows/deploy-credits-sandbox.yml

```diff
@@ -21,6 +21,7 @@ jobs:
             product_lifetime_in_days: 14
             default_credits_per_user: 0
             reset_credits_monthly: true
+            cost_profile: EDC_UAT
```

Comment:
`EDC` should be sufficient here, since I expect we'll ultimately use the same label for both EDC environments